hack/make.ps1

```diff
@@ -353,19 +362,13 @@ Function Run-IntegrationTests() {
         $pinfo = New-Object System.Diagnostics.ProcessStartInfo
         $pinfo.FileName = "gotestsum.exe"
         $pinfo.WorkingDirectory = "$($PWD.Path)"
-        $pinfo.RedirectStandardError = $true
         $pinfo.UseShellExecute = $false
-        $pinfo.Arguments = "--format=standard-quiet --jsonfile=$jsonFilePath --junitfile=$xmlFilePath -- $env:INTEGRATION_TESTFLAGS"
+        $pinfo.Arguments = "--format=standard-verbose --jsonfile=$jsonFilePath --junitfile=$xmlFilePath -- $env:INTEGRATION_TESTFLAGS"
         $p = New-Object System.Diagnostics.Process
         $p.StartInfo = $pinfo
         $p.Start() | Out-Null
         $p.WaitForExit()
-        $err = $p.StandardError.ReadToEnd()
-        if (($LASTEXITCODE -ne 0) -and ($err -notlike "*warning: no tests to run*")) {
-            Throw "Integration tests failed: $err"
```

Comment:
IIUC, both stdout and stderr will now be printed directly instead of buffered because you removed `$pinfo.RedirectStandardError = $true`, so no longer needed to manually print the output correct?

Reply:
I noticed that gotestsum actually prints errors also on stdout instead of stderr so that was unnecessary redirection and because I now use `$p.ExitCode` there is no need to have that old logic anymore which I added there because of way how `go test` was printing warnings.